server/src/main/java/de/llggiessen/mke/controller/BookingController.java

```diff
@@ -70,5 +70,19 @@
         return repository.findAllByStatus(status);
     }
 
+    @DeleteMapping(value = "", params = {"id"})
+    public void deleteById(@RequestParam long id){
+        repository.deleteById(id);
+    }
+
+    @PostMapping("")
+    public Booking createBooking(@RequestBody Booking booking){
+        try {
+            return repository.save(booking);
```

Comment:
save() by default overrides existing entry if primary key (id) is the same. This behavior should not be default when doing a POST request and only possible when doing a PUT.